### PR TITLE
Add support for treating date entries as type `datetime`

### PR DESCRIPTION
**Pull request by GiaJordan**
Convert manifest string entries for dates into type `datetime` during csv import as dataframe
Addresses #800 only, in the future will add support for storing date information on synapse, work on the type as used in schematic, add validation rules, etc. as appropriate

### CODE REVIEW

**Comment by linglp** [comment]
@GiaJordan I want to change the description of endpoint `/get/datatype/manifest` along with this PR. I will let you know when I finish that. Thanks for your patience!

**Comment by GiaJordan** [comment]
@linglp no problem! thanks for the heads up!

**Review by linglp — APPROVED** [review]

File: schematic/utils/df_utils.py [hunk 1]
```diff
@@ -45,11 +49,25 @@ def load_df(file_path, preserve_raw_input=True, data_model=False, **load_args):
         #Trim nans and empty rows and columns
         processed_df = trim_commas_df(float_df)
         
-        #Store values that were entered as ints
+        #Store values that were entered as ints and dates
         processed_df=processed_df.mask(ints != False, other = ints)  
 
+        #dates = processed_df.applymap(lambda x: _parse_dates(x), na_action='ignore').fillna(False)
```
Comment (linglp) [inline schematic/utils/df_utils.py:55]: @GiaJordan Need to be deleted?

Comment (GiaJordan) [reply schematic/utils/df_utils.py]: yes, thanks for catching!

**Review by andrewelamb — APPROVED** [review]

**Pull request merged**